templates/archives/stories/women_in_symbols.jinja2

```diff
@@ -0,0 +1,28 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+Women in Symbols
+{% endblock %}
+
+{% block story_body %}
+<p>It is often quite difficult to find the work of women in our society’s historical record, particularly in regards to computing. Most of the historical materials we reference to learn about early computing have been signed by men, with little evidence of the women behind them. For example, a majority of the letters in this archive have been written by a secretary, whose initials only appear at the end of the letter, such as in the picture to the right, where FMV has signed the letter and NB has written it up.</p>
+
+<figure class="figure float-right">
+    <img src="{{ static('img/stories/women_in_symbols.png') }}"
+         class="class="figure-img img-fluid rounded"
+         alt="A letter subscript above the cc line detailing those involved in its writing" />
+</figure>
+
+<p>However, internal documents sometimes display a portion of the work women performed in
+    computing. In the case of the choices made for which symbols computers would use, this
+    archive demonstrates that women were heavily involved. In a memo submitted to Laura by Edna
```

Comment:
I believe there were also women working as programmers in these spaces, if not here then in other labs. If we don't have this evidence in our archive, maybe we should focus on these particular documents to show that women were involved in decision making and correspondence.